setup.py

```diff
@@ -5,8 +5,14 @@
 
 # Get a list of scripts to install
 scripts = []
-for root, _, filenames in os.walk('scripts'):
+for root, dirnames, filenames in os.walk('scripts'):
```

Comment:
That is the default value in all Python versions: https://docs.python.org/2/library/os.html#os.walk https://docs.python.org/3/library/os.html#os.walk